_layouts/api.html

```diff
@@ -34,24 +34,33 @@ <h2>
   {% endif %}
 </p>
 
-<p>
+
   {% if page.customisation.size > 0 %}
-  <a href="/customisation">Customisation</a> is supported.
-  {% endif %}
-  {% if page.customisation contains 'Glossary' %}
-    <a href="/glossaries">Glossaries</a> can be used to customise the translation of specific words and phrases.
+    <p>
+      <a href="/customisation">Customisation</a> is supported.
+      {% if page.customisation contains 'Glossary' %}
+        <a href="/glossaries">Glossaries</a> can be used to customise the translation of specific words and phrases.
+      {% endif %}
+      {% if page.customisation contains 'Formality' %}
+        The API can also be customised for <a href="/formality">formality</a>.
+      {% endif %}
+      {% if page.customisation contains 'Fine-tuning' %}
+        <a href="/fine-tuning">Fine-tuning</a> on <a href="/parallel-data">parallel data</a> can be used to fine-tune the model.
+      {% endif %}
+      {% if page.customisation contains 'Adaptive' %}
+        The API also supports <a href="/adaptive">adaptive</a> customisation.
+      {% endif %} <br>
+      {% if page.file_translation %}
+        The API also supports <a href='/file-translation'>file translation</a>.
+      {% endif %}
+    </p>
   {% endif %}
-  {% if page.customisation contains 'Formality' %}
-    The API can also be customised for <a href="/formality">formality</a>.
-  {% endif %}
-  {% if page.customisation contains 'Fine-tuning' %}
-    <a href="/fine-tuning">Fine-tuning</a> on <a href="/parallel-data">parallel data</a> can be used to fine-tune the model.
-  {% endif %}
-  {% if page.customisation contains 'Adaptive' %}
-    The API also supports <a href="/adaptive">adaptive</a> customisation.
-  {% endif %} <br>
-  {% if page.file_translation %}
-    The API also supports <a href='/file-translation'>file translation</a>.
+
+<p>
+  {% if page.prompt_required == true %}
+    The <strong>{{ page.title }}</strong> API requires a prompt for translation.
```

Comment:
The <strong>{{ page.title }}</strong> API is <strong>not translation-specific</a>, so for translation it <strong>requires a prompt</a>.